### PR TITLE
MDEV-32535 Update signal hander user info more compassion and correct url

### PR DESCRIPTION
**Pull request by grooverdan**
- [x] *The Jira issue number for this PR is: MDEV-32535*

## Description

Blaming hardware and poor libraries seems on the rare end of the scale of things that go wrong. Accept the blame, apologize, and offer some ascii art, and kindly request a bug report.

Also url change on stack traces is changed to include mariadbd.

Thanks Vlad for also raising that blaming was wrong.

## How can this PR be tested?

Ask a user how they feel about the previous message compared to the new one.

## Basing the PR against the correct MariaDB version
- [ ] *This is a new feature and the PR is based against the latest MariaDB development branch.*
- [X] *This is a bug fix and the PR is based against the earliest maintained branch in which the bug can be reproduced.*

## PR quality check
- [X] I checked the [CODING_STANDARDS.md](https://github.com/MariaDB/server/blob/11.0/CODING_STANDARDS.md) file and my PR conforms to this where appropriate.
- [X] For any trivial modifications to the PR, I am ok with the reviewer making the changes themselves.

### CODE REVIEW

**Comment by illuusio** [comment]
As I'm thumbs up with this. What is the license of that nice bug ASCII art?

File: sql/signal_handler.cc [hunk 1]
```diff
@@ -170,11 +170,13 @@ extern "C" sig_handler handle_fatal_signal(int sig)
   my_safe_printf_stderr("[ERROR] mysqld got " SIGNAL_FMT " ;\n",sig);
 
   my_safe_printf_stderr("%s",
-    "This could be because you hit a bug. It is also possible that this binary\n"
-    "or one of the libraries it was linked against is corrupt, improperly built,\n"
-    "or misconfigured. This error can also be caused by malfunctioning hardware.\n\n");
-
+                        "Sorry, we probably make a mistake, and this is a bug.\n"
```
Comment (vaintroub) [inline sql/signal_handler.cc:173]: English is not my native language, but should not that be "made a mistake",  in the past tense?

Comment (grooverdan) [reply sql/signal_handler.cc:173]: yep it shold be.

**Comment by vuvova** [comment]
I don't think ASCII art in the log is a particularly good idea. Logs are mainly line-oriented append-only files which are nowadays consumed, parsed, and processed by various log management and analysis tools. Pretty pictures inside the log are just garbage that needs to be ignored.

**Review by vaintroub — APPROVED** [review]
Looks good

**Pull request merged**